fusionauth/resource_fusionauth_themes_test.go

```diff
@@ -210,6 +210,8 @@ func generateFusionAuthTemplate() fusionauth.Templates {
 		Oauth2Register:                            randString20(),
 		Oauth2StartIdPLink:                        randString20(),
 		Oauth2TwoFactor:                           randString20(),
+		Oauth2TwoFactorEnable:                     randString20(),
```

Comment:
Based on update of themes update from https://github.com/gpsinsight/terraform-provider-fusionauth/pull/216 to make unit tests runnable